web/components/application/hacker/pages/PageOne/PageOneConstraints.js

```diff
@@ -40,6 +40,13 @@ export const constraints = {
   email: {
     email: true,
   },
+  confirmEmail: {
```

Comment:
I think we might need to add `presence: true` here, the next button is enabled for me when I don't fill in `confirmEmail` field